Makefile

```diff
@@ -208,9 +226,9 @@ GOTEST_PARALLEL ?= '-test.parallel=4'
 
 localTestEnv:
 	bin/testEnvLocalK8S.sh ensure
-	go install istio.io/istio/pilot/test/server
-	go install istio.io/istio/pilot/test/client
-	go install istio.io/istio/pilot/test/eurekamirror
+	go build -o ${GOPATH}/bin/server istio.io/istio/pilot/test/server
```

Comment:
Can we give these better names than `server` and `client`, given we're dumping them into the GOPATH's bin (which is usually on the user's PATH). `pilots` and `pilotc` to mirror Mixer's binary names, maybe?

Reply:
pilot-test-server, pilot-test-client ..